src/CheckoutSdk/Payments/Links/PaymentLinksClient.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System.Threading;
+using System.Threading.Tasks;
+
+namespace Checkout.Payments.Links
+{
+	public class PaymentLinksClient : AbstractClient, IPaymentLinksClient
+	{
+
+		private const string PaymentLinksPath = "payment-links";
+
+		public PaymentLinksClient(IApiClient apiClient, CheckoutConfiguration configuration) : base(apiClient, configuration, SdkAuthorizationType.SecretKey)
+		{
+		}
+
+		public Task<PaymentLinkDetailsResponse> GetAsync(string reference, CancellationToken cancellationToken = default)
+		{
+			CheckoutUtils.ValidateParams("reference", reference);
+			return ApiClient.Get<PaymentLinkDetailsResponse>(PaymentLinksPath + "/" + reference, SdkAuthorization(), cancellationToken);
```

Comment:
Please use `BuildPath()` from the abstract client. Like this: `BuildPath(PaymentLinksPath, reference)`.